pinc/User.inc

```diff
@@ -2,6 +2,7 @@
 include_once($relPath.'misc.inc');
 include_once($relPath.'access_log.inc');
 include_once($relPath.'SettingsClass.inc');
+include_once($relPath.'../api/exceptions.inc');
```

Comment:
I'm not loving how the API exceptions are being pulled into `Project.inc` and `User.inc`. The API should use those classes / files not the other way around. I need to ponder this some more because I see _why_ you did it.

Reply:
Yes, it would involve lots of throwing and catching.